### PR TITLE
(feat) Added accessibilityLabels to SegmentedControlTab.

### PR DESCRIPTION
**Pull request by jayzamazing**
This change allows star rating to be targeted by Appium for automation.

### CODE REVIEW

**Comment by saikrishna321** [comment]
@kirankalyan5  why hasn't this been merged or reviewed?

**Comment by SrinivasanTarget** [comment]
@kirankalyan5 Not maintaining this repo actively?

File: SegmentedControlTab.js [hunk 1]
```diff
@@ -34,14 +34,17 @@ const TabOption = ({
             firstTabStyle,
             lastTabStyle]}
             onPress={() => onTabPress(index)}
-            activeOpacity={1}>
-            <View style={{ flexDirection: "row" }}>
+            activeOpacity={1}
+            accessibilityLabel={accessibilityLabel + "_touchableOpacity"}>
```
Comment (saikrishna321) [inline SegmentedControlTab.js]: should be: accessibilityLabel={accessibilityLabel}  

@kirankalyan5  Thoughts?

Comment (kirankalyan5) [reply SegmentedControlTab.js]: True, so we give the liberty for the consumer to set the `accessibilityLabel`. 

This issue will be soon fixed as a part of #55 PR.

**Comment by kirankalyan5** [comment]
Closing this issue and will be tracked as a part of #55

**Comment by kirankalyan5** [comment]
Closing this issue and will be tracked as a part of #55

**Pull request closed without merging**